src/plugins/unified_search/public/query_string_input/text_based_languages_editor/helpers.test.ts

```diff
@@ -51,42 +51,6 @@ describe('helpers', function () {
       ]);
     });
 
-    it('should return the correct error object if dataview not found for an one liner query', function () {
```

Comment:
ℹ️ We removed this error (dataview not found) as we now create an adhoc instead. So these tests are not necessary now.

Reply:
I love it when we get to delete things